### PR TITLE
FIX: random.gumbel segfault on GPU

### PR DESCRIPTION
**Pull request by samir-nasibli**
# Description
* fix `random.gumbel` segfault on GPU
temporary solution for #569

### CODE REVIEW

File: dpnp/backend/kernels/dpnp_krnl_random.cpp [hunk 1]
```diff
@@ -308,15 +311,35 @@ void dpnp_rng_gumbel_c(void* result, const double loc, const double scale, const
...
+    {
+        event = mkl_blas::scal(DPNP_QUEUE, size, alpha, result1, incx);
+    }
+    else
+    {
+        // for (size_t i = 0; i < size; i++) result1[i] *= alpha;
+        cl::sycl::range<1> gws(size);
+        auto kernel_parallel_for_func = [=](cl::sycl::id<1> global_id) {
+            size_t i = global_id[0];
+            {
```
Comment (shssf) [inline dpnp/backend/kernels/dpnp_krnl_random.cpp:334]: D owe really need these extra brackets?

File: dpnp/backend/kernels/dpnp_krnl_random.cpp [hunk 2]
```diff
@@ -298,6 +298,9 @@ void dpnp_rng_geometric_c(void* result, const float p, const size_t size)
     event_out.wait();
 }
 
+template <typename _KernelNameSpecialization>
+class _dpnp_blas_scal_c_kernel;
```
Comment (shssf) [inline dpnp/backend/kernels/dpnp_krnl_random.cpp:302]:
```suggestion
class dpnp_blas_scal_c_kernel;
```

File: dpnp/backend/kernels/dpnp_krnl_random.cpp [hunk 3]
```diff
@@ -308,15 +311,35 @@ void dpnp_rng_gumbel_c(void* result, const double loc, const double scale, const
...
+        // for (size_t i = 0; i < size; i++) result1[i] *= alpha;
+        cl::sycl::range<1> gws(size);
+        auto kernel_parallel_for_func = [=](cl::sycl::id<1> global_id) {
+            size_t i = global_id[0];
+            {
+                result1[i] *= alpha;
+            }
+        };
+        auto kernel_func = [&](cl::sycl::handler& cgh) {
+            cgh.parallel_for<class _dpnp_blas_scal_c_kernel<_DataType>>(gws, kernel_parallel_for_func);
```
Comment (shssf) [inline dpnp/backend/kernels/dpnp_krnl_random.cpp:339]:
```suggestion
            cgh.parallel_for<class dpnp_blas_scal_c_kernel<_DataType>>(gws, kernel_parallel_for_func);
```

**Comment by samir-nasibli** [comment]
@shssf @Alexander-Makaryev all tests, waiting approve.

**Review by Alexander-Makaryev — APPROVED** [review]
I suggest to keep issue https://github.com/IntelPython/dpnp/issues/569 in open state to be able to track and revert these changes when fix in third party components will be provided.

**Review by shssf — APPROVED** [review]
no need to use `std::` in `cpp`

**Pull request merged**